packages/applicant/src/pages/mandatory-questions/start.page.tsx

```diff
@@ -44,6 +45,22 @@ export async function getServerSideProps({
       },
     };
   }
+  try {
+    const applicationStatus = await getApplicationStatusBySchemeId(
+      schemeId,
+      jwt
+    );
+    if (applicationStatus === 'REMOVED') {
+      return {
+        redirect: {
+          destination: `/service-error?serviceErrorProps={"errorInformation":"This application has been unpublished.","linkAttributes":{"href":"/","linkText":"Go back to the home page","linkInformation":""}}`,
+          permanent: false,
+        },
+      };
+    }
+  } catch (error) {
```

Comment:
Feel like we shouldn't be throwing errors for normal app behaviour. Can we handle this any better?

Reply:
This is due to the endpoint for fetching the applications status throwing an error if no application exists. However this is now valid app behaviour.

Could refactor this endpoint to not throw an error, and instead return a new status of `EXTERNAL_APPLICATON`?